.eleventy.js

```diff
@@ -15,15 +15,12 @@ module.exports = function(eleventyConfig) {
     // Put robots.txt in root
     eleventyConfig.addPassthroughCopy({ 'src/robots.txt': '/robots.txt' });
 
-    // TODO: when we move to 11ty v1.0, use the following
-    // rather than ignoring git ignore
-    //eleventyConfig.ignores.delete("src/handbook");
+    eleventyConfig.ignores.delete("src/handbook");
 
     eleventyConfig.addLayoutAlias('default', 'layouts/base.njk');
     eleventyConfig.addLayoutAlias('page', 'layouts/page.njk');
     eleventyConfig.addLayoutAlias('nohero', 'layouts/nohero.njk');
     eleventyConfig.addLayoutAlias('redirect', 'layouts/redirect.njk');
-    eleventyConfig.addPassthroughCopy("src/images");
```

Comment:
Note that the regexp 5 lines below will catch this. ** might also be empty in that regexp